distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1308,45 +1310,53 @@ type Table
     write_json : File.File -> Nothing
     write_json file = this.to_json.to_text.write file
 
-    ## UNSTABLE
+    ## This function writes a table from memory into a file.
 
-       Writes the table to a specified file with the given serialization
-       settings.
+       The specific behavior of the various `File_Format`s is specified below.
 
        Arguments:
-       - file: the file to write to.
-       - format: the format settings to use.
-
-
-       > Example
```

Comment:
I added it but seems that currently the `Excel` format does not seem to have an option for writing the headers?

I'm also unsure about the defaults - because currently the default for Excel format is `Sheet_Names` which is a sensible default for reading, but does not make much sense for writing.